File: PHP/Booking/DiscountInformation.php
<?php
namespace LTD_API;

class DiscountInformation 
{
    // array of DiscountItem
    private $discounts = array();
    
    public function getDiscounts()
    {
        return $this->discounts;
    }
    public function setDiscounts($discounts)
    {
        $this->discounts = $discounts;
    }
}

File: Java/Booking/DiscountInformation.java
package LTD_API;

import java.util.List;

public class DiscountInformation
{
    private List<DiscountItem> discounts;
    
    public List<DiscountItem> getDiscounts()
    {
        return discounts;
    }
    public void setDiscounts(List<DiscountItem> discounts)
    {
        this.discounts = discounts;
    }
}

